Applications/Games/Space Colony/Local/script.js

```diff
@@ -12,13 +12,13 @@ new LocalInstallerScript()
     .wineDistribution("upstream")
     .wineVersion(getLatestDevelopmentVersion)
     .executable("Space Colony.exe")
-    .postInstall(function (wine /*wizard*/) {
+    .postInstall((wine) => {
         const patch = new Resource()
             .wizard(this._wizard)
             .url("https://d1ztm8591kdhlc.cloudfront.net/hdpatches/Space_Colony_HD_Update.exe")
             .checksum("c821e5c7035b9b517823466f4cedadd3")
             .algorithm("MD5")
             .name("Space_Colony_HD_Update.exe")
             .get();
-        wine.run(patch);
+        wine.run(patch, null, false, true);
```

Comment:
This is a general comment:

Do we want to force the script developers to provide all these parameters or do we want to allow for a shorthand version `wine.run(patch);`? According to my findings it seems like we still have a lot of scripts that use the shorthand form

Reply:
That depends on what those parameters do and if any of them can crash the exe file in question.